docs/en/latest/tutorials/add-multiple-api-versions.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Add multiple API versions
+keywords:
+  - API Versioning
+  - Multiple APIs
+  - Proxy rewrite
+  - Request redirect
+  - Route API requests
+description: In this tutorial, you will learn how to _publish and manage multiple versions of your API_ with Apache APISIX.
```

Comment:
We can probably remove the "_" as description is more of a metadata and it would not work semantically.